.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -34,6 +34,19 @@
           find . -path "**/build/reports/*" -or -path "**/build/test-results/*" > artifacts.list
           rsync -R --files-from=artifacts.list . ${{ github.job }}-artifacts
           tar -zcvf ${{ github.job }}-jdk${{ matrix.jdk }}-logs.tar.gz ${{ github.job }}-artifacts
+      - name: Generate Test Reports
+        if: success() || failure()
```

Comment:
What does this do? Is it the equivalent of `if (true || false)`? Or are there more states besides success and failure, for which we would want to avoid generating a report...?

Reply:
By default GitHub workflow will skip if the previous steps fail. By specifying both it means to run regardless of the outcome of the previous steps.